packages/material-ui/src/RadioGroup/RadioGroup.js

```diff
@@ -96,7 +106,7 @@ RadioGroup.propTypes = {
    */
   children: PropTypes.node,
   /**
-   * The default input value, useful when not controlling the component.
+   * The required default input value of an uncontrolled RadioGroup.
```

Comment:
I would suggest the following:
- In 90% of the cases, users will be better off with a select radio value, no matter what.
- NN/g encourages to always have a selected radio: https://www.nngroup.com/articles/checkboxes-vs-radio-buttons/.
- But, it's not something everybody agrees with, [e.g. Windows](https://docs.microsoft.com/fr-fr/windows/desktop/uxguide/ctrl-radio-buttons#default-values). They recognize that an always select value should be the norm. However, there are exceptions where not providing a default value is more appropriate:
> Exceptions: Don't have a default selection if:
> 1. There is no acceptable default option for safety, security, or legal reasons and therefore the user must make an explicit choice. If the user doesn't make a selection, display an error message to force one.
> 2. The user interface (UI) must reflect the current state and the option hasn't been set yet. A default value would - incorrectly imply that the user doesn't need to make a selection.
> 3. The goal is to collect unbiased data. Default values would bias data collection.
> 4. The group of radio buttons represents a property in a mixed state, which happens when displaying a property for multiple objects that don't have the same setting. Don't display an error message in this case since each object has a valid state.

So, I'm in favor of encouraging an always selected radio, but allow people to have zero selected.